delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/ResourcesTrialRoutes.scala

```diff
@@ -63,8 +63,8 @@ final class ResourcesTrialRoutes(
     pathPrefix("resources") {
       extractCaller { implicit caller =>
         resolveProjectRef.apply { project =>
-          authorizeFor(project, Write).apply {
```

Comment:
Where it was placed made that in the end only users with `resources/write` could access the resources